examples/seismic/tutorials/10_nmo_correction.ipynb

```diff
@@ -291,6 +291,7 @@
     }
    ],
    "source": [
+    "#NBVAL_IGNORE_OUTPUT\n",
```

Comment:
THere is an increasing number of nbval as we discussed.
What if we add some norm checking in some places?
Here we could check the norms of rec

Reply:
>THere is an increasing number of nbval as we discussed.

These are on figure cells. the test/asserts shoulds still all be fine as they are in their own (at least they should be)

Reply:
OK, we good then